mint/build/admin/subnet-health-usage.go

```diff
@@ -0,0 +1,57 @@
+/*
+*
+*  Mint, (C) 2021 Minio, Inc.
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+ */
+
+package main
+
+import (
+	"context"
+	"time"
+)
+
+// testSubnetHealthUsage tests that the usage data (bucket count, object count and usage)
+// is present (and non-zero) in the output of the healthinfo api. It is assumed that the
+// test always runs against a cluster with non-zero usage.
+func testSubnetHealthUsage() {
+	function := "testSubnetHealthUsage"
+	startTime := time.Now()
+	args := map[string]interface{}{}
+
+	si, err := adminClient.ServerInfo(context.Background())
+	if err != nil {
+		failureLog(function, args, startTime, "", "Subnet health server info failed", err).Fatal()
+		return
+	}
+
+	if si.Buckets.Count == 0 {
```

Comment:
This is normal on a fresh setup - its not instantaneous

Reply:
Don't we run the tests against `play` that always has some data? As I mentioned [here](https://github.com/minio/minio/pull/11945#issuecomment-815446819), if there is a possibility of the tests running against an fresh/empty cluster, then I'll have to add the code to create test bucket(s), put object(s) in them, and also wait for some time (how much?) before performing the test.

Reply:
No @anjalshireesh we don't we are running this against every PR that is submitted, this test will always fail and it shall be flaky. - This test will make sense perhaps in future when we add real-time usage calculation.

Reply:
> Don't we run the tests against `play` that always has some data? As I mentioned [here](https://github.com/minio/minio/pull/11945#issuecomment-815446819), if there is a possibility of the tests running against an fresh/empty cluster, then I'll have to add the code to create test bucket(s), put object(s) in them, and also wait for some time (how much?) before performing the test.

No adding such arbitrary waits are not possible and unexpected - I don't think we can add such tests that do not have a predefined time window to complete.

Reply:
Thanks @harshavardhana , I was under the wrong assumption that the test will always run against a cluster that has some data in it (and is not a fresh cluster). Since that assumption is wrong and adding an arbitrary wait is not a good idea either, looks like I'll have to abandon this PR.

OR

Maybe I'll think about some other test(s) we can add related to `mc admin subnet health` and update this PR with the same.

cc @kannappanr